dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/Skills/GitHubSkill/openapi.json

```diff
@@ -0,0 +1,5935 @@
+{
```

Comment:
Is there a URL we could get this from instead of checking in our own copy?

Reply:
afaik no, GH just has one massive specification for all of their APIs so I had to reduce this manually: https://github.com/github/rest-api-description/tree/main/descriptions